src/reformat.c

```diff
@@ -621,9 +621,9 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
                 B = Y;
             }
 
-            R = AVIF_CLAMP(R, 0.0f, 1.0f);
-            G = AVIF_CLAMP(G, 0.0f, 1.0f);
-            B = AVIF_CLAMP(B, 0.0f, 1.0f);
```

Comment:
I considered simply moving these three lines after the if statement, to replace lines 664-666 below. Unfortunately, that may potentially allow a large value of R, G, or B to be multiplied by an A less than but very close to 1.0f at lines 645-647 and then be clamped to exactly 1.0f. That seems incorrect. So I didn't do that.